commands.js

```diff
@@ -1187,7 +1187,7 @@ var commands = exports.commands = {
 			target = target.substr(0, commaIndex);
 		}
 		targetUser = Users.get(target);
-		if (!targetUser) return this.sendReply("User '" + this.targetUsername + "' not found.");
+		if (!targetUser) return this.sendReply("User '" + (targetUser ? targetUser.name : target) + "' not found.");
```

Comment:
Aha, culprit was 3ea6fe03cf. Ironic, since it was intended to _improve_ /fr messages...